client/blocks/app-banner/index.jsx

```diff
@@ -258,15 +220,13 @@ export function buildDeepLinkFragment( currentRoute, currentSection ) {
 const mapStateToProps = ( state ) => {
 	const sectionName = getSectionName( state );
 	const isNotesOpen = isNotificationsOpen( state );
-	const currentRoute = getCurrentRoute( state );
```

Comment:
This line removal seems to have broken things. `currentRoute` is undefined now.

Reply:
Thank you for noticing that, it was left out after the rebase